motor_controller/odriver/script/odriver_s1_node.py

```diff
@@ -324,6 +331,35 @@ def _error_recovery(relaunch = True):
         if (_odrv_has_error(odrvs[0]) and relaunch) or (_odrv_has_error(odrvs[1]) and relaunch) :
             _relaunch_odrive()
 
+# Refer to https://roboticsbackend.com/ros2-rclpy-parameter-callback/
+def parameters_callback(params):
+    global vel_gain, vel_integrator_gain
+    success = False
+    for param in params:
+        if param.name == "vel_gain":
+            if param.type_ in [Parameter.Type.DOUBLE, Parameter.Type.INTEGER]:
+                if param.value >= 0.0 and param.value < 100.0:
+                    try:
+                        odrvs[0].axis0.controller.config.vel_gain = param.value
+                        odrvs[1].axis0.controller.config.vel_gain = param.value
```

Comment:
Is there no concern about concurrent access to parameters?

Reply:
Thank you for the review. See https://github.com/CMU-cabot/cabot-drivers/pull/26#discussion_r1741458307